citesphere/src/main/webapp/WEB-INF/views/auth/group/items.html

```diff
@@ -13,7 +13,15 @@
 	    visiblePages: 10,
 	    initiateStartPageClick: false,
 	    onPageClick: function (event, page) {
-	    	window.location.href = "items?page=" + page;
+	        var currentURL = window.location.href;
+	        if (currentURL.match(/search/)) {
+	            var key = $("#search").val();
+	            if (key != undefined && key != '') {
+	                window.location.href="search?searchTerm=" + key + "&page=" + page;
+	            }
+	        } else {
+	            window.location.href = "items?page=" + page;
+	        }
```

Comment:
I would take all query parameter and add them to the url, otherwise if we extend the functionality of the page and add more query params, we will have to change this again.

Reply:
?

Reply:
I am not sure if any other parameter is applicable over here. I have added sort by. Only others are "collectionId" and "index" which does not seem to apply.

Reply:
yes, but in the future we might add functionality that adds more parameters

Reply:
what do you mean by "both of them"?

Reply:
collectionId and index